server/datastore/mysql/mysql.go

```diff
@@ -987,7 +987,11 @@ func registerTLS(conf config.MysqlConfig) error {
 // provided configuration.
 func generateMysqlConnectionString(conf config.MysqlConfig) string {
 	params := url.Values{
-		"charset":              []string{"utf8mb4"},
+		// using collation implicitly sets the charset too
+		// and it's the recommended way to do it per the
+		// driver documentation:
+		// https://github.com/go-sql-driver/mysql#collation
+		"collation":            []string{"utf8mb4_general_ci"},
```

Comment:
:+1: Just a heads-up, the mention of the "recommended way" is in the charset section: https://github.com/go-sql-driver/mysql#charset